src/launchers/linux.ts

```diff
@@ -2,8 +2,17 @@ import execa from "execa";
 import { Launcher } from "./";
 
 export const linux: Launcher = (execFilePath, options) => {
+  let argArray = ["-e", "{{command}}"];
+  if (options.terminalAppSetup !== undefined) {
+    argArray = options.terminalAppSetup.split(" ");
+  }
+  for (let i = 0; i < argArray.length; i++) {
+    if (argArray[i] === "{{command}}") {
+      argArray[i] = `sh ${execFilePath}`;
+    }
+  }
```

Comment:
It would be nice to see a test for this feature, we do run the tests on linux and a test shouldn't be too hard to add.